docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -41,3 +41,86 @@ the delegating client by default under the hood, so this can be now removed.
 - The functions `fake.NewFakeClientWithScheme` and `fake.NewFakeClient` have been deprecated.
 - Switch to `fake.NewClientBuilder().WithObjects().Build()` instead, which provides a cleaner interface
   to create a new fake client with objects, lists, or a scheme.
+
+## Move the webhooks back in the main manager
+
+**Pre-requisites**
+
+Provider's `/config` folder has the same structure of  `/config` folder in CAPI controllers.
```

Comment:
What does this mean?

Reply:
This instruction assumes the provider's /config folder has the same structure we are maintaining in CAPI (I'm not sure all the providers have the same structure)

Reply:
should this say as the /config folder in v1alpha3?